.install/python-macro.yaml

```diff
@@ -0,0 +1,163 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >-
+  This CFN Template allows you to install the AWS CFN Macro as a Lambda Function to execute ECS Compose-X directly
+  from within your AWS CloudFormation templates.
+
+Transform:
+  - AWS::Serverless-2016-10-31
+
+Metadata:
+  Maintainer: john@compose-x.io
+  Version: 2021-01-25
+  AWS::CloudFormation::Interface:
+    ParmeterLabels:
+      ComposeXLambdaLayerVersion:
+        default: ECS Compose-X Macro Lambda Layer version
+      TemplatesBucket:
+        default: Bucket to store all rendered CloudFormation templates and config files.
+    ParameterGroups:
+      - Parameters:
+          - MemorySize
+        Label:
+          default: Lambda Settings
+      - Parameters:
+          - TemplatesBucket
+        Label:
+          default: ECS ComposeX Settings
+
+
+Parameters:
+  ComposeXLambdaLayerVersion:
+    Type: String
+    AllowedPattern: (arn:aws([a-z]+)?:lambda:[a-z0-9-]+:\d{12}:layer:[a-zA-Z0-9-_]+:\d{1,3}$)
+    Default: arn:aws:lambda:eu-west-1:518078317392:layer:ecs_composex:3
+  MemorySize:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Lambda Memory Size parameters should use MinValue, MaxValue or AllowedValues at Parameters/MemorySize